packages/caip/src/caip19/caip19.ts

```diff
@@ -19,7 +19,8 @@ export enum AssetNamespace {
 export enum AssetReference {
   Bitcoin = 0,
   Ethereum = 60,
-  Cosmos = 118
+  Cosmos = 118,
+  Osmosis = 118
```

Comment:
Anyway we can close and re-open this PR to remove some of the github diff noise?